synfig-studio/src/gui/docks/dock_timetrack2.cpp

```diff
@@ -37,6 +37,7 @@
 #include <gui/widgets/widget_timetrack.h>
 #include <gui/canvasview.h>
 #include <gui/localization.h>
+#include <gui/app.h>

```

Comment:
i included it so that i can use the already implemnted functions in app.h for deleting the popup menu.
`delete_widget()` and  what it leads to which `really_delete_widget()`

Reply:
removed in new commit, no longer needed